recipes/strangepg/meta.yaml

```diff
@@ -1,17 +1,18 @@
 {% set name = "strangepg" %}
-{% set version = "0.8.13" %}
+{% set version = "0.8.14" %}
+{% set ref = "1e666b80f464f9e898e8fce1e42f96eaf10953be" %}
```

Comment:
This commit is a hotfix. I want to avoid setting another tag and triggering a new autobump which will also fail. The next manual bump will restore tag-based releases.